set.mm

```diff
@@ -527093,6 +527118,180 @@ element of the complex projective line (Riemann sphere) to its inverse.
+       precise definition is irrelevant and should generally not be used.
+       (New usage is discouraged.) $)
+    df-bj-iomnn $a |- iomnn = ( ( n e. _om |->
+        <. [ <. { r e. Q. | r <Q <. suc n , 1o >. } , 1P >. ] ~R , 0R >. ) u.
+                                                    { <. _om , pinfty >. } ) $.
+  $}
+
+  ${
+    $d x A $.  $d x F $.  $d x G $.
+    $( If the direct image of a singleton under any of two functions is the
+       same, then the values of these function at the corresponding point
+       agree.  (Contributed by BJ, 18-Mar-2023.) $)
+    bj-imafv $p |-
+                  ( ( F " { A } ) = ( G " { A } ) -> ( F ` A ) = ( G ` A ) ) $=
+      ( vx csn cima wceq cv cab cuni cfv eqeq1 abbidv unieqd dffv4 3eqtr4g ) BA
+      EZFZCQFZGZRDHEZGZDIZJSUAGZDIZJABKACKTUCUETUBUDDRSUALMNDABODACOP $.
+  $}
+
+  ${
+    bj-funun.un $e |- ( ph -> F = ( G u. H ) ) $.
+    bj-funun.neldm $e |- ( ph -> -. A e. dom H ) $.
+    $( Value of a function expressed as a union of two functions at a point not
+       in the domain of one of them.  (Contributed by BJ, 18-Mar-2023.) $)
+    bj-funun $p |- ( ph -> ( F ` A ) = ( G ` A ) ) $=
+      ( csn cima wceq cfv cun imaeq1 imaundir syl6eq syl c0 cdm wcel wn ndmima
+      uneq2 un0 eqtrd bj-imafv ) ACBHZIZDUFIZJBCKBDKJAUGUHEUFIZLZUHACDELZJZUGUJ
+      JFULUGUKUFIUJCUKUFMDEUFNOPAUIQJZUJUHJABERSTUMGBEUAPUMUJUHQLUHUIQUHUBUHUCO
+      PUDBCDUEP $.
+  $}
+
+  ${
+    bj-fununsn.un $e |- ( ph -> F = ( G u. { <. B , C >. } ) ) $.
+    ${
+      bj-fununsn1.neq $e |- ( ph -> -. A = B ) $.
+      $( Value of a function expressed as a union of a function and a singleton
+         on a couple (with disjoint domain) at a point not equal to the first
+         component of that couple.  (Contributed by BJ, 18-Mar-2023.)
+         (Proof modification is discouraged.) $)
+      bj-fununsn1 $p |- ( ph -> ( F ` A ) = ( G ` A ) ) $=
+        ( cop csn cdm wss dmsnopss a1i wceq wcel elsni nsyl ssneldd bj-funun )
+        ABEFCDIJZGAUAKZCJZBUBUCLACDMNABCOBUCPHBCQRST $.
+    $}
+
+    ${
+      bj-fununsn2.neldm $e |- ( ph -> -. B e. dom G ) $.
+      bj-fununsn2.ex1 $e |- ( ph -> B e. V ) $.
+      bj-fununsn2.ex2 $e |- ( ph -> C e. W ) $.
+      $( Value of a function expressed as a union of a function and a singleton
+         on a couple (with disjoint domain) at the first component of that
+         couple.  (Contributed by BJ, 18-Mar-2023.)
+         (Proof modification is discouraged.) $)
+      bj-fununsn2 $p |- ( ph -> ( F ` B ) = C ) $=
+        ( cfv cop csn cun uncom syl6eq bj-funun wcel wceq fvsng syl2anc eqtrd )
+        ABDLBBCMNZLZCABDUDEADEUDOUDEOHEUDPQIRABFSCGSUECTJKBCFGUAUBUC $.
+    $}
+  $}
+
+  $( An element of a difference with a singleton is not equal to the element of
+     that singleton.  Note that ` ( -. A e. { C } -> -. A = C ) ` need not hold
+     if ` A ` is a proper class.  (Contributed by BJ, 18-Mar-2023.) $)
+  eldifsnneq $p |- ( A e. ( B \ { C } ) -> -. A = C ) $=
```

Comment:
Still a theorem not following your naming style

Reply:
Oh, yes. I plan to move it to Main soon anyway, so it can stay like that.

Reply:
This is, in principle, the same as ~eldifsni. I think we do not need both in main.

Reply:
Oh, I see, the MM-PA didn't catch it when I tried to prove the other theorem, so I added it without looking further.